packages/components/src/components/toggle-button/toggle-button.css

```diff
@@ -0,0 +1,191 @@
+/**
+ * @license
+ * Scale https://github.com/telekom/scale
+ *
+ * Copyright (c) 2021 Egor Kirpichev and contributors, Deutsche Telekom AG
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+:host {
+  --width: auto;
+  --spacing-x: var(--scl-spacing-24);
+  --spacing-x-icon-only: var(--scl-spacing-8);
+  --min-height: var(--scl-spacing-24);
+  --xs-height: var(--scl-spacing-24);
+  --small-height: var(--scl-spacing-32);
```

Comment:
the pattern we are trying to follow is: `--cssProperty-thing-state`
so like the following:
```
--height-small
--height-regular
...
```

Reply:
css naming has beed adapted...